### PR TITLE
Invoice logo updated at centre position #6345

### PR DESCRIPTION
**Pull request by papnoisanjeev**
## Issue Reference
#6345

## Description
Updated invoice logo position to centre.

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
Please divide the shipping details and invoice details with separate-separate section, Please take a reference from [Here](https://prnt.sc/QACscJIy4VJD)

**Review by Sanjaybhattwebkul — APPROVED** [review]

File: packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php [hunk 1]
```diff
@@ -117,6 +117,12 @@
             .text-center {
                 text-align: center;
             }
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php:119]: Add new line here

File: packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php [hunk 2]
```diff
@@ -125,76 +131,77 @@
...
+                        @endif
                         <h1 class="text-center">{{ __('admin::app.sales.invoices.invoice') }}</h1>
                     </div>
                 </div>
+                <div class="row">
+                    <div class="col-12">
+                        <div class="col-6">
+                            <div class="merchant-details">
+                                <div>
+                                    <span class="merchant-details-title">{{ core()->getConfigData('sales.shipping.origin.store_name') ? core()->getConfigData('sales.shipping.origin.store_name') : '' }}</span>
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php:147]: Change this as below

`core()->getConfigData('sales.shipping.origin.store_name') ?? '''`

File: packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php [hunk 3]
```diff
@@ -125,76 +131,77 @@
...
-                    <div class="image">
-                        <img class="logo" src="{{ Storage::url(core()->getConfigData('sales.invoice_setttings.invoice_slip_design.logo')) }}" alt=""/>
-                    </div>
-                @endif
-
-                <div class="merchant-details">
-                    <div>
-                        <span class="merchant-details-title">{{ core()->getConfigData('sales.shipping.origin.store_name') ? core()->getConfigData('sales.shipping.origin.store_name') : '' }}</span>
-                    </div>
+                                <div>{{ core()->getConfigData('sales.shipping.origin.address1') ? core()->getConfigData('sales.shipping.origin.address1') : '' }}</div>
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php:150]: `core()->getConfigData('sales.shipping.origin.address1') ?? ''`

File: packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php [hunk 4]
```diff
@@ -125,76 +131,77 @@
...
 
-                    <div>{{ core()->getConfigData('sales.shipping.origin.address1') ? core()->getConfigData('sales.shipping.origin.address1') : '' }}</div>
+                                <div>
+                                    <span>{{ core()->getConfigData('sales.shipping.origin.zipcode') ? core()->getConfigData('sales.shipping.origin.zipcode') : '' }}</span>
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php:153]: Null coalescing operator can be used here



<br class="Apple-interchange-newline">

File: packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php [hunk 5]
```diff
@@ -125,76 +131,77 @@
...
+                                    <span>{{ core()->getConfigData('sales.shipping.origin.city') ? core()->getConfigData('sales.shipping.origin.city') : '' }}</span>
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php:154]: Null coalescing operator can be used here

File: packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php [hunk 6]
```diff
@@ -125,76 +131,77 @@
...
+                                </div>
 
-                    <div>
-                        <span>{{ core()->getConfigData('sales.shipping.origin.zipcode') ? core()->getConfigData('sales.shipping.origin.zipcode') : '' }}</span>
-                        <span>{{ core()->getConfigData('sales.shipping.origin.city') ? core()->getConfigData('sales.shipping.origin.city') : '' }}</span>
-                    </div>
+                                <div>{{ core()->getConfigData('sales.shipping.origin.state') ? core()->getConfigData('sales.shipping.origin.state') : '' }}</div>
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php:157]: Null coalescing operator can be used here

File: packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php [hunk 7]
```diff
@@ -125,76 +131,77 @@
...
 
-                    <div>{{ core()->getConfigData('sales.shipping.origin.state') ? core()->getConfigData('sales.shipping.origin.state') : '' }}</div>
+                                <div>{{ core()->getConfigData('sales.shipping.origin.country') ?  core()->country_name(core()->getConfigData('sales.shipping.origin.country')) : '' }}</div>
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php:159]: Null coalescing operator can be used here

Comment (papnoisanjeev) [reply packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php]: done

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Comment by devansh-webkul** [comment]
I am closing this pull request as no need of adding the logo to the center as an address is already placed below the logo.

**Pull request closed without merging**